diff --git a/lib/index.js b/lib/index.js
index c3e5fba44f3568a0de1f1b54bb1e54ee6773f657..9c6a5d068f794fb6646716fcf26547821453b63c 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -49,7 +49,7 @@ module.exports = function(){
   return generator
 }
 
-Generator = function(options = {}){
+const Generator = function(options = {}){
   // Convert Stream Readable options if underscored
   if(options.high_water_mark){
     options.highWaterMark = options.high_water_mark
